integration-test/data/FoD/fod-integation-test-data.json

```diff
@@ -0,0 +1,221 @@
+[
+  {
+    "commands": "fod session login",
+    "arguments": "",
+	"waitSeconds": "2"
+  },
+  {
+    "commands": "fod session list",
+    "arguments": "",
+	"expectOutput": "default"
+  },  
+  {
+    "commands": "fod application create-web-app",
+    "arguments": "--owner=1851 --business-criticality=High --sdlc-status='Production' INT_T_App2:INT_T_App2_Release2",
+	"expectOutput": "created"

```

Comment:
What about any mandatory attributes that might be required in the different tenants, We need to think how this would be done.